docs/source/software/quick_start_guide.rst

```diff
@@ -108,5 +108,33 @@ QGroundControl and change the following:
 * EKF2_HGT_MODE = vision
 
 
-Safety  
-=======================
\ No newline at end of file
+Safety Checklist
+==================
+
+Make sure you complete the following check list *before every flight*. Most of the checks are automated by the controller on startup, but that is no substitute for good ole human caution.
```

Comment:
I would make this part a warning or important directive, so that it is highlighted. You should be able to do this with the following:

```
.. important::
    Make sure you ...
```

Reply:
done

Reply:
This looks great with the directive!